Tools/CISupport/ews-build/steps.py

```diff
@@ -3533,6 +3538,17 @@ def commandComplete(self, cmd):
 
     def setLayoutTestCommand(self):
         super(RunWebKitTestsWithoutChange, self).setLayoutTestCommand()
+        if self.getProperty('github.base.ref', DEFAULT_BRANCH) == DEFAULT_BRANCH:
```

Comment:
Maybe we should also add a condition like: `if CURRENT_HOSTNAME == EWS_BUILD_HOSTNAME` (we don't want to report from uat instance or local testing instances)

Reply:
Not sure if the check is necessary. Perhaps in the future we'd want to report from the UAT instance to results-dev.safari.apple.com for testing purposes?